### ISSUE TITLE
Can someone explain to me what the 2 mask nodes do in the "PuLID_attention_mask.json" example workflow?

### ISSUE THREAD

**Issue by ClothingAI** [cubiq/PuLID_ComfyUI#95]
![image](https://github.com/user-attachments/assets/c808b96b-0df0-4bfd-9a26-52c6ac7740cc)

I am not sure what these nodes do exaclty here in this example from this repo examples folder?

**Comment by cubiq** [comment]
they are used to create a mask so we can apply two different pulid models to each half

**Comment by ClothingAI** [comment]
So it draw automaticlly 2 rectangles, right?
So instead of having to draw them manually with the mouse (for example) it pre "render" 2 areas for you?

**Comment by cubiq** [comment]
exactly

**Comment by ClothingAI** [comment]
But they are randomly placed am I wrong? I don't undertand fully the usefulness of it^^.
